root/static/scripts/common/constants/locales.json

```diff
@@ -607,14 +634,24 @@
   "rw_RW":"Kinyarwanda Rwanda",
   "rwk":"Rwa",
   "rwk_TZ":"Rwa Tanzania",
+  "sa":"Sanskrit",
+  "sa_IN":"Sanskrit India",
   "sah":"Sakha",
   "sah_RU":"Sakha Russia",
   "saq":"Samburu",
   "saq_KE":"Samburu Kenya",
+  "sat":"Santali",
+  "sat_Olck":"Santali Ol Chiki",
+  "sat_Olck_IN":"Santali India Ol Chiki",
   "sbp":"Sangu",
   "sbp_TZ":"Sangu Tanzania",
+  "sc":"Sardinian",
+  "sc_IT":"Sardinian Italy",
   "sd":"Sindhi",
-  "sd_PK":"Sindhi Pakistan",
```

Comment:
We had one use of this one for artists that needed amending, changed it to just Sindhi.

Reply:
It seems that `sd_Arab_PK` might be more appropriate, as planned in comment to the linked ticket.

Reply:
I assumed that meant it should be in Arabic script :)

Reply:
I assumed that is the only script used in Pakistan, but that might just be the only `sd_*_PK` locale with a good coverage.